tests/unit/conftest.py

```diff
@@ -118,11 +121,63 @@ def test_dir(tmpdir_factory) -> str:
     return str(tmp_path)
 
 
-@pytest.fixture(scope="session")
-def config(test_dir) -> Config:
-    """A good/known Config object"""
-    return Config(
-        f"{test_dir}/pipeline.yaml",
-        f"{test_dir}/input_data.yaml",
-        f"{test_dir}/environment.yaml",
+@pytest.fixture()
+def default_config_params(test_dir) -> Dict[str, str]:
```

Comment:
I'd skip the indirection and just pass this dict into `dummy_config()` in the `default_config` fixture below